docs/users.md

```diff
@@ -1,6 +1,8 @@
 # Users
 
-This is a collection of user methods. 
+This is a collection of method that are intended to be helpful wrappers around the [Users api endpoints](user-api-docs).
+
+The definition of the [User response object can be found here](user-object-reference).
```

Comment:
How about:

```
View the [User response object](user-object-reference) for details on the properties you'll get back with each response
```

Reply:
Nice, I like that better. It felt odd since I have to link them to a specific endpoint to get the object reference.